akka-cluster/src/test/scala/akka/cluster/ResetSystemMessageSeqNrSpec.scala

```diff
@@ -22,66 +23,68 @@ class ResetSystemMessageSeqNrSpec extends ArteryMultiNodeSpec("""
   akka.cluster.jmx.multi-mbeans-in-same-jvm = on
   """) with ImplicitSender {
 
-  "System messages sequence numbers" should {
+  // This test is not for classic remoting
+  if (RARP(system).provider.remoteSettings.Artery.Enabled)
```

Comment:
these two lines is the only change, rest is indentation

Reply:
This class extends `ArteryMultiNodeSpec` - if we could do something there to make sure it's only ran when Artery is enabled that seems even better. But if that's tricky this is :+1: as well obv.

Reply:
Updated to use `withFixture` instead